.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -0,0 +1,158 @@
+- type: textarea
+  id: justification
+  attributes:
+    label: Justification
+    description: Why you believe this issue meets the [cherry-pick criteria](https://github.com/ampproject/amphtml/blob/main/docs/contributing-code.md#Cherry-picks).
+    placeholder: Provide a specific rationale for why this fix cannot wait until the next Stable release.
+  validations:
+    required: true
+- type: textarea
+  id: why_not_beta_experimental
+  attributes:
+    label: Why Not Beta / Experimental?
+    description: Why a Stable cherry-pick is necessary, but not Beta / Experimental.
+    placeholder: If the cherry-pick is going into Stable but not Beta / Experimental, provide a rationale (e.g. the PR is already part of the beta channel). Otherwise, skip this section.
+- type: textarea
+  id: why_lts
+  attributes:
+    label: Why LTS?
+    description: Why an LTS cherry-pick is necessary.
+    placeholder: If the cherry-pick is also going into LTS, provide a rationale (e.g. the bug is visible to LTS users). Otherwise, skip this section.
+- type: textarea
+  id: verification_steps
+  attributes:
+    label: Verification Steps
+    description: How the fix can be verified once the cherry-pick has been performed.
+    placeholder: Provide detailed steps to manually verify the changes in this cherry-pick. They will be run by a developer or a QA expert to ensure that the bug was indeed fixed.
+  validations:
+    required: true
+- type: textarea
+  id: cherry_pick_progress
+  attributes:
+    label: Cherry-Pick Progress
+    description: Progress details for this cherry-pick.
+    value: |
+      <!-- Leave this section as it is for now. -->
+
+      To be updated by @ampproject/release-on-duty as each stage is completed.
+      - [x] Cherry-pick request created
+      - [ ] Cherry-pick request approved
+      - [ ] New version released to Beta / Experimental channels
+      - [ ] New version released to Stable channel
+      - [ ] New version released to LTS channel
+      - [ ] Release tracker updated
+      - [ ] Cherry-pick completed
+- type: markdown
+  id: mini_postmortem
+  attributes:
+    value: |
+      ## Mini Postmortem
+      The following sections serve as a postmortem for this cherry-pick, and are meant to document things like the bug summary, its root causes, user impact, and action items. Fill in these sections to the best of your knowledge so that similar issues can be prevented in future.
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary
+    description: A summary of the problem and its root cause(s).
+    placeholder: |
+      Provide answers to questions like:
+      - What is the bug being fixed?
+      - What are its root cause(s)?
+      - How did it end up getting released to stable channel?
+      If details are as yet unknown, put down "TODO" in this section and remember to fill it in later.
+  validations:
+    required: true
```

Comment:
Still not convinced that these fields should be mandatory because 
- PMs aren't required for Beta CPs
- Time is of the essence when filing these requests. These fields will almost always have TODOs, but how will we remind people to come back to fill it in?

Open to being convinced!

Reply:
> These fields will almost always have TODOs, but how will we remind people to come back to fill it in?

These fields are currently empty in roughly half of all CP requests, which indicates that users either did not see the section (due to the wall of text), ignored it, or forgot about it. My gut feeling is that a more readable / usable form will result in a higher response rate.

> PMs aren't required for Beta CPs

This is a good point, and I think it has pushed me over the fence for this PR. I've made the post-mortem fields optional, and adjusted the text so that it's clear that Beta CPs can skip. 

If future evidence suggests that we actually need to switch from opt-in to opt-out to improve response rates, or that our team's post-mortem policy necessitates that these fields be mandatory, we can come in and make the change separately.

_Filed under #well-argued-point #esther-was-right_

Reply:
SGTM :)